### ISSUE TITLE
Quick-docs (expandable inline help) not available for <img> tags

### ISSUE THREAD

**Issue by flukeout** [mozilla/thimble.webmaker.org#671]
They seem to be available for all other tags, but if you try to open the quickdocs (Command / Control + K) on https://bramble.mofostaging.net/?enableExtensions=WebPlatformDocs for the ```<img />``` tag you get an error.

**Comment by humphd** [comment]
Upstream issue, we need to update the `html.json` used by that extension, see https://github.com/adobe/brackets/pull/10036#issuecomment-123761479

**Comment by humphd** [comment]
We need to pickup the change happening upstream, when it lands.

**Comment by humphd** [comment]
I thought this would be fixed when I merged the most recent Brackets stuff, but it's still an issue.  We should maybe consider switching over to use our MDN docs extension.

**Comment by humphd** [comment]
...which is https://github.com/mozilla/brackets/tree/master/src/extensions/extra/MDNDocs

**Comment by gideonthomas** [comment]
don't think MDN docs is localized though

**Comment by Pomax** [comment]
does it need to be? Doesn't MDN automatically tap into your locale already?

**Comment by humphd** [comment]
We probably don't scrape from all localized versions.  https://github.com/mozilla/brackets/blob/master/src/extensions/extra/MDNDocs/tools/scrape-mdn/app.js#L19 seems to confirm that.  I bet we can fix that.

**Comment by gideonthomas** [comment]
Yeah, as @humphd mentions, we use a scraper since MDN does not have an API that you can use. Also, I spent a bunch of time trying to get that fixed when I was localizing Thimble. It really is a rabbit hole. Different locales on MDN have different data formats and structures and scrapping them is a nightmare because there is very little consistency (because MDN is a Wiki). 

I honestly don't think it is worth figuring that out on our own (but I think Brackets might be switching over to it since WebPlatformDocs maintenance has been discontinued so maybe they might be able to figure out how to get localized MDNDocs).

One suggestions is that we can switch to using MDNDocs but disable docs for localized Brackets. Might be the easier solution.

**Comment by Pomax** [comment]
I just noticed https://developer.mozilla.org/en-US/docs/MDN/About#Using_MDN_content ... do we attribute in our quick docs?

**Comment by gideonthomas** [comment]
We don't currently attribute since we don't actually use the plugin. If we do, we definitely should add that attribution.

**Comment by humphd** [comment]
Let's wait on this to see how things pan out upstream, and maybe add the `img` stuff manually ourselves.
